tests/features/rhpam/businesscentral/rhpam-businesscentral.feature

```diff
@@ -61,3 +61,29 @@ Feature: RHPAM Business Central configuration tests
      And container log should contain -Dorg.uberfire.nio.git.http.enabled=true
      And container log should contain -Dorg.uberfire.nio.git.http.hostname=example.com
      And container log should contain -Dorg.uberfire.nio.git.http.port=80
+
+  Scenario: Verify if the properties were correctly set using DEFAULT MEM RATIO
+  Scenario: Verify if the properties were correctly set using DEFAULT MEM RATIO
```

Comment:
Remove one of two scenario lines for businesscentral
```suggestion
  Scenario: Verify if the properties were correctly set using DEFAULT MEM RATIO
```

Reply:
Fixed